hack/jenkins/e2e.sh

```diff
@@ -231,13 +231,19 @@ echo "Test Environment:"
 printenv | sort
 echo "--------------------------------------------------------------------------------"
 
-if [[ "${E2E_UP,,}" == "true" ]]; then
+# We get the kuberntes tarballs on either cluster creation or when we want to
+# replace existing ones in a multi-step job (e.g. a cluster upgrade).
+if [[ "${E2E_UP,,}" == "true" || ${JENKINS_FORCE_GET_TARS:-} =~ ^[yY]$ ]]; then
```

Comment:
At least fix the newly introduced variables. Up to you if you want to fix the rest of the file as well.